generate-formula.sh

```diff
@@ -0,0 +1,89 @@
+# Check if version and binary are provided
+if [ -z "$1" ] || [ -z "$2" ]; then
+    echo "Usage: $0 <binary> <version>"
+    exit 1
+fi
+
+BINARY=$1
```

Comment:
I would call it BINARY_NAME as binary can be more things (my first thought was that you are requireing file before looking into the code)